src/styles/main.scss

```diff
@@ -1,3 +1,244 @@
+$card-with: 200px;
+$color-accent: #00acdc;
+$margin-right-card: 50px;
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+h2,
+h3,
+p {
+  margin: 0;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  color: #{$color-accent};
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  min-width: 924px;
+  padding: 0 50px;
+  text-transform: uppercase;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.img {
+  display: block;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    padding: 23px 0;
+    color: #060b35;
+    text-decoration: none;
+    align-content: center;
+    transition: all 0.3s;
+
+    &:hover {
+      color: #{$color-accent};
+    }
+
+    &:active {
+      position: relative;
+      color: #{$color-accent};
+      display: block;
+    }
+
+    &:active::after {
+      position: absolute;
+      display: block;
+      content: "";
+      height: 4px;
+      width: 100%;
+      right: 0;
+      left: 0;
+      bottom: 0;
+      background-color: #{$color-accent};
+      border-radius: 8px;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  color: #{$color-accent};
+  display: block;
+}
+
+.is-active::after {
+  position: absolute;
+  display: block;
+  content: "";
+  height: 4px;
+  width: 100%;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background-color: #{$color-accent};
+  border-radius: 8px;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.